README.md

```diff
@@ -995,6 +995,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Wahyu Wiyoko](https://github.com/wahyuwiyoko)
   - [Waqe](https://github.com/waq-e)
   - [Willi Hohenstein](https://github.com/willhoh)
+  -[William J Priest](https://github.com/WilliamJPriest)
```

Comment:
```suggestion
  - [William J Priest](https://github.com/WilliamJPriest)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
oh okay, let me fix that